conda/common/os/__init__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from logging import getLogger
+
+from ..os.linux import linux_get_libc_version
+from ..os.unix import get_free_space_on_unix, is_admin_on_unix
+from ..os.windows import get_free_space_on_windows, is_admin_on_windows
+from ..compat import on_win
+
+
+log = getLogger(__name__)
+
+
+linux_get_libc_version
```

Comment:
We shouldn't be importing from all platforms all the time here.  Should be more like

```python
if on_win:
    from .windows import is_admin_on_windows as is_admin
else:
    from .unix import is_admin_on_unix as is_admin
```

We don't need the `linux_get_libc_version` import at all in this module I don't think.